osde2e/managed_cluster_validating_webhooks_test.go

```diff
@@ -165,7 +194,7 @@ var _ = Describe("Managed Cluster Validating Webhooks", Ordered, func() {
 
 			err := client.Get(ctx, saName, namespaceName, sa)
 
-			if err == nil {
+			if (err == nil) {
```

Comment:
This is not necessary, should be left as is 👍

Reply:
@Amarthya-v Please make sure that your editor is using tools like `gofmt` which will catch stuff like this.